packages/ui/src/Button.tsx

```diff
@@ -1,83 +1,58 @@
 import React, { ButtonHTMLAttributes, FC } from 'react'
 import cn from 'classnames'
-import { Icon } from 'react-feather'
 
+import { Icon, IconProps } from './Icon'
 import { TruncatedText } from './TruncatedText'
 
 import styles from './Button.module.scss'
 
-export type ButtonKind = 'primary' | 'secondary'
-/* 
-    | 'success'
-    | 'successSecondary'
-    | 'info'
-    | 'infoSecondary'
-    | 'warning'
-    | 'warningSecondary'
-    | 'dashed'
-    | 'critical'
-    | 'criticalSecondary'
-    | 'danger'
-   */
+export type ButtonKind = 'primary' | 'secondary' | 'transparent'
 
 // Left icon is always present with proper aria-label attribute
 type AccessibleIconLeftProps =
   | {
-      IconLeft: Icon
+      iconLeft: IconProps['icon']
       iconLeftAriaLabel: string
+      iconLeftSize?: IconProps['size']
+      iconLeftColor?: IconProps['color']
+      iconLeftClassName?: string
```

Comment:
nit: Do you think we have a use-case for these extra props right now? Our buttons at this point are of fixed height and colors. So the icons are also of fixed height and colors.

Reply:
These are defined because we need to tweak the spacing a bit for Alerts actions: https://github.com/hazelcast/frontend-shared/pull/49/files#diff-674337aa6f682273f1632c6b0625a49e0606aa8f8e06facd4ca62a475ed76231R86.

Reply:
Oh, I see. We not only decreased the margin there, but also decreased the font size of the button. Do you think it makes sense to add a `size` prop to the `Button` with possible values `'small' | 'normal'`. For the `small` `Button` we could decrease the margins, change the icon sizes and remove capitalizations. In other words, kind of encapsulate these styling changes. WDYT?

Reply:
I think I am basically building on the previous idea we discussed during the initial `Button` implementation. Instead of having lots of props with lots of freedom, we should try created several more or less rigid presets. WDYT?

Reply:
Conceptually, that makes sense to me, however are we sure that we'll ever use `small` button in any case expect from this one? I would defer this if we have another use-case.

Reply:
Fair enough. Let's not add potentially semi-dead code.